src/views/Ticket.js

```diff
@@ -112,9 +122,16 @@ class Ticket extends React.Component {
           <Button
             className={'mt-8'}
             prop-size={'lg wide'}
+            disabled={this.state.isUnlocking || !Maybe.Nothing.hasInstance(wallet)}
```

Comment:
Just a couple of notes re: `folktale/maybe` use:

* It's probably better to use `Maybe.Just.hasInstance(foo)` instead of `!Maybe.Nothing.hasInstance(foo)`.  I know there are a few instances of `!Maybe.Nothing.hasInstance` floating around from the last rush of code pre-initial release, but we should avoid introducing more of them.

* I'd also started to import folktale/maybe via `import { Just, Nothing } from ..` in recent contributions, which cuts down on a lot of the verbose `Maybe.`'s that pop up everywhere needlessly.

(Of course, if anyone wants to just rip this whole library out, go for it.  But I do find it very useful -- especially when its verbosity issues are addressed.)

Reply:
Fixed them here and elsewhere, too.